src/platform/api.py

```diff
@@ -87,3 +89,13 @@ def get_last_edited_user_name(row):
     else:
         user_id = row.edited_by_user_drn.split(":")[-1]
     return get_user_name(user_id)
+
+
+@beartype
+def get_secrets() -> list[dict]:
```

Comment:
In most places we call this "variables and secrets" because some are encrypted and some are not. I suggest aligning the terminology in the method name.

Reply:
fixed